client/blocks/login/index.jsx

```diff
@@ -45,17 +46,27 @@ export class Login extends Component {
 	}
 
 	onChangeField( event ) {
-		this.setState( {
-			[ event.target.name ]: event.target.value
-		} );
+		const target = event.target;
+		const value = target.type === 'checkbox' ? target.checked : target.value;
+		const name = target.name;
+
+		if ( name === 'rememberme' ) {
+			this.props.recordTracksEvent( 'calypso_loginblock_rememberme_change', { new_value: value } );
+		}
+
+		this.setState( { [ name ]: value } );
 	}
```

Comment:
@drewblaisdell: just curious, I see you added a commit with my proposal but you opted to bind the new event handler explicitly in the constructor:

```javascript
this.onChangeRememberMe = this.onChangeRememberMe.bind( this );
```

@gziolo introduced me this week with using an arrow function as a way to avoid that additional line of code, something that I used above in my proposal:

```javascript
onRememberMeChange = ( event ) => {
  ...
};
```

Do you see a problem with that approach? Or did you simply want to keep the code consistent in this class, since we bind some other methods explicitly in the constructor?